cypress.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = defineConfig({
     admin_username: 'admin',
     admin_password: '',
     test_site_section: '/about-cancer/understanding',
-    front_end_base_url: 'https://www-dev-ac.cancer.gov'
+    front_end_base_url: 'https://www-test-acsf.cancer.gov'
```

Comment:
revert this change - it should not be committed, you only change values to test locally and revert back after